tools/distrib/python/grpcio_tools/setup.py

```diff
@@ -230,7 +230,7 @@ def extension_modules():
     ext_modules=extension_modules(),
     packages=setuptools.find_packages('.'),
     install_requires=[
-        'protobuf>=3.5.0.post1, < 4.0dev',
+        'protobuf=={}'.format(grpc_version.PROTOBUF_VERSION),
```

Comment:
ProtoBuf is falling behind in recent binary wheel releasing. They don't release for Linux 32bit, 3.8 on Windows and [3.9](https://github.com/protocolbuffers/protobuf/issues/7978). This PR's change might be blocked by them. I would recommend to find another more tolerant solution.

Reply:
Good point. I have some alternative ideas to address this that still require `grpcio-tools` to know what version of `protobuf` it was compiled against. For the moment, I'm just going to remove the `install_requires` change. This PR will still give us some nice automation for the next time we update protobuf, which should be happening roughly every gRPC release.

Reply:
btw, they don't yet release source wheels, so users don't have the option to compile themselves.